website/docs/configurations.md

```diff
@@ -21,6 +21,7 @@ It helps to have a central configuration file for your common cross job configur
 
 By default, Hudi would load the configuration file under `/etc/hudi/conf` directory. You can specify a different configuration directory location by setting the `HUDI_CONF_DIR` environment variable.
 - [**Spark Datasource Configs**](#SPARK_DATASOURCE): These configs control the Hudi Spark Datasource, providing ability to define keys/partitioning, pick out the write operation, specify how to merge records or choosing query type to read.
+- [**Parquet Configs**](#PARQUET_CONFIG): These configs makes it possible to bring native parquet features
```

Comment:
The `configurations.md` is automatically generated by `https://github.com/apache/hudi/tree/asf-site/hudi-utils` so let's avoid changing this page directly.